src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,57 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/tab';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+}
+
+export const Tabs: React.FC<Props> = (props) => {
+  const {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  } = props;
+
+  const handleSelectedTab = (tab: Tab) => {
+    if (tab.id === selectedTabId) {
+      return;
+    }
+
+    onTabSelected(tab);
+  };
```

Comment:
`
 if (tab.id !== selectedTabId) {
       selectedOnTab(tab);
     }
`

here we don't need to use two conditions